azure-iot-device/samples/advanced-hub-scenarios/register_x509.py

```diff
@@ -23,17 +23,11 @@
 
 async def main():
     async def register_device():
-        x509_cert = X509(
-            cert_file=os.getenv("X509_CERT_FILE"), key_file=os.getenv("X509_KEY_FILE")
-        )
-        x509_security_client = X509SecurityClient(
+        x509_cert = X509(cert_file=os.getenv("X509_CERT_FILE"), key_file=os.getenv("X509_KEY_FILE"))
+        provisioning_device_client = ProvisioningDeviceClient.create_from_x509_certificate(
             provisioning_host, registration_id, id_scope, x509_cert
```

Comment:
>x509_cert [](start = 58, length = 9)

nit: this is probably not a cert anymore. it's a certificate + key (+ passphrase potentially)

Reply:
just renamed to x509

---
In reply to: [294543995](https://github.com/Azure/azure-iot-sdk-python-preview/pull/109#discussion_r294543995) [](ancestors = 294543995)